src/NewTwitchApi/Resources/UsersApi.php

```diff
@@ -73,6 +73,23 @@ public function getUsersFollows(string $followerId = null, string $followedUserI
             $queryParamsMap[] = ['key' => 'after', 'value' => $after];
         }
 
-        return $this->callApi('users/follows', $queryParamsMap);
+        return $this->callApi('users/follows', $queryParamsMap, $bearer);
+    }
+
+    /**
+     * @throws GuzzleException
+     * @link https://dev.twitch.tv/docs/api/reference/#get-broadcaster-s-subscribers
+     */
+    public function getBroadcasterSubscriptions(string $broadcasterId, array $ids = [], string $bearer): ResponseInterface
```

Comment:
Thank you for adding this! In order to keep up with the code structure, please move this into a new `SubscriptionsApi` class in this same namespace. 

The way I'm organizing these is using the table at the top of https://dev.twitch.tv/docs/api/reference/ with the resource being the class name and the endpoint being this method. Since "Get Broadcaster's Subscribers" is in the "Subscriptions" resource, we should keep the code structured to reflect that, to remain consistent.

Reply:
@echosa Thanks for your reply. I've made this adjustment as requested. I hope it meets your requirements. In the coming days, I'll work on extending the Subscription API to complete it for you. 

That table was a great help. I appreciate your time.

Brandin.